### DISCUSSION TITLE
Coordinator acting up.

### DISCUSSION THREAD

**Discussion by domgggg** [MikaylaFischler/cc-mek-scada#426] [Support] [answered]
When I try to startup the coordinator it always says on the main monitor "supervisor cooling configuration invalid, check supervisor config file" I checked the config file and I cant find anything to do with "cooling configuration".

Also when I go into the config file for the rtu, it shows up way different then how it looks in your videos. The lines of code you show arent in the config file.

**Comment by MikaylaFischler** [comment, marked as answer]
Did you configure the coordinator? It gives that message if the coordinator unit count doesn't match the supervisor unit count. The next update will reword that message.

The initial setup guide video (the long one) is very out of date. Please refer to the newer videos for updated configuration. Here is the RTU: https://www.youtube.com/watch?v=8Rk860eRMH4

The coordinator's updated configuration tool will be coming out today or tomorrow, along with a video for it.

**Reply by domgggg** [reply]
Thanks the monitors are working now.
